src/main/scala/cromwell/binding/expression/Evaluator.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding.expression
+
+import cromwell.parser.WdlParser.AstNode
+
+import scala.language.postfixOps
+import scala.util.Try
+
+trait Evaluator {
+  type T
```

Comment:
I think this is perhaps the place in their style guide that the typesafe folks are the least consistent, they're not even consistent in their inconsistencies :) I wager it has to do w/ which libraries are dominated by java folks vs the haskell crowd

My personal preference is what you're describing, single meaningful capital letter. My order of personal desire is:

Single meaningful capital letter > A > T > WholeWord >>> WHOLE_WORD >>>> multiple meaningful capital letters

That last one is way down there :)